### DISCUSSION TITLE
Add a way to disable having the buildOptions.site url prepended to the link value of an item in an rss feed

### DISCUSSION THREAD

**Discussion by edazpotato** [withastro/roadmap#40] [Proposal]
This is very important if, for example, a feed includes links to off-site articles or posts that the site owner finds interesting.

**Comment by FredKSchott** [comment]
Can you give an example of what you are seeing? Are you saying that remote URLs currently aren't supported in an RSS feed?

**Reply by edazpotato** [reply]
Yeah, they get prefixed with the value in `buildOptions.site`.
E.g. if `buildOptions.site` was `https://edaz.codes` the link `https://gist.github.com/edazpotato/whatever` would become `https://edaz.codes/gist.github.com/edazpotato/whatever` in the rss feed
